holidays/countries/yemen.py

```
@@ -0,0 +1,207 @@
+        if self._year >= 2000:
+            # Prophet's Birthday.
+            self._add_mawlid_day(tr("المولد النبوي"))
+
+            # Isra' and Mi'raj.
+            self._add_isra_and_miraj_day(tr("ذكرى الإسراء والمعراج"))
+
+            # Victory Day.
+            self._add_holiday_jul_7(tr("ذكرى 7 يوليو"))
+
+
+class YE(Yemen):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)